resources/assets/js/models/types.ts

```diff
@@ -172,12 +172,26 @@ export interface Job {
   culture_special: localizedField;
 }
 
+export interface JobApplicationAnswer {
+  id: number;
+  job_poster_questions_id: number;
+  job_application_id: number;
+  answer: localizedFieldNonNull;
+}
+
 export interface JobPosterKeyTask {
   id: number;
   job_poster_id: number;
   description: localizedFieldNonNull;
 }
 
+export interface JobPosterQuestion {
+  id: number;
+  job_poster_id: number;
+  question: localizedFieldNonNull;
+  description: localizedFieldNonNull;
```

Comment:
These fields ARE nullable, so use `localizedField` instead. Generally, most content entered by a user (including a manager) can be null. `localizedFieldNonNull` should mostly be used for constants that are only created by devs or admins, like Skills or Provinces.